File: Enkle algoritmer/twoSum/twoSum.java
import java.util.Arrays;

public class twoSum{
    public static void main(String[] args){
        int[] tallListe = {4,1,12,2,6,5,2,7,11,8,3};

        System.out.println(twoSumAlgo(tallListe));
    }

    public static String twoSumAlgo(int[] tall) {
        int[] output = {};
        int target = 9;

        for(int i=0; i<tall.length; i++){
            for(int j=0; j<tall.length; j++){
                if(tall[i]+tall[j] == target){
                    output = Arrays.copyOf(output, output.length+2);
                    output[output.length - 2] = tall[i];
                    output[output.length - 1] = tall[j];
                }
            }
        }
        return Arrays.toString(output);
    }
}

File: Enkle algoritmer/twoSum/twoSum.c
#include <stdio.h>

int main(){
    int tallListe[4] = {2,3,7,6};
    int target = 9;
    printf("%s", twoSum());
}

int twoSum(){
    int tallListe[4] = {2,3,7,6};
    int target = 9;
    int output[] = {};

    for(int i=0; i<4; ++i){
        for(int j=0; j<4; ++j){
            if(tallListe[i] + tallListe[j] == target){
                output[0] = tallListe[i];
                output[1] = tallListe[j];
            }
        }
    }
    return output;
}



// int main(){
//     int x = 48;
//     while (x<50){
//         printf("X is less than 50. ");
//         x+=1;
//         if(x >= 50){
//             printf("X is greater or equal than 50. ");
//             break;
//         }
//     }
// }